aggregator_core/src/batch_mode.rs

```diff
@@ -418,90 +412,3 @@ impl CollectableBatchMode for LeaderSelected {
             .await
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use crate::{
-        batch_mode::CollectableBatchMode,
-        task::{AggregationMode, BatchMode, test_util::TaskBuilder},
-    };
-    use janus_core::vdaf::VdafInstance;
-    use janus_messages::{
-        Duration, Interval, Time, batch_mode::TimeInterval, taskprov::TimePrecision,
-    };
-
-    #[test]
-    fn validate_collect_identifier() {
```

Comment:
Should we still test that collection intervals smaller than the task's time precision get rejected?